roaring/btree.go

```diff
@@ -876,6 +876,42 @@ func (e *enumerator) Next() (k uint64, v *Container, err error) {
 	return k, v, err
 }
 
+// Every iterates over a tree.
+func (e *enumerator) Every(upd func(oldV *Container, exists bool) (newV *Container, write bool)) error {
+	if err := e.err; err != nil {
```

Comment:
Copying the idiom from Next and Prev, which are probably doing it because possibly in the past they were using naked returns with the named parameters? In any event, I'm inclined to keep the idiom parallel with the upstream ones.

I'm not sold on this function. I sort of want to make it take a func returning an error, and yield the error, but that makes it incompatible with the update funcs used with Put.